site/en/docs/android/custom-tabs/guide-warmup-prefetch/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Warm-up and pre-fetching: using the Custom Tabs Service"
```

Comment:
The items before the colon need to be the same part of speech: either "Warming-up and pre-fetching" or "Warm-up and pre-fetch".

Reply:
fixed